packages/client.api/src/index.ts

```diff
@@ -25,6 +25,34 @@ export type {
   ApplyBatchActionOptions,
   OsdkActionParameters,
 } from "./actions/Actions.js";
+export type { AggregatableKeys } from "./aggregate/AggregatableKeys.js";
+export type { AggregateOpts } from "./aggregate/AggregateOpts.js";
+export type { AggregateOptsThatErrors } from "./aggregate/AggregateOptsThatErrors.js";
+export type { AggregationResultsWithGroups } from "./aggregate/AggregationResultsWithGroups.js";
+export type {
+  AggregationCountResult,
+  AggregationResultsWithoutGroups,
+} from "./aggregate/AggregationResultsWithoutGroups.js";
+export type { AggregationClause } from "./aggregate/AggregationsClause.js";
+export type { AggregationsResults } from "./aggregate/AggregationsResults.js";
+export type {
+  AndWhereClause,
+  GeoFilter,
+  GeoFilter_Intersects,
+  GeoFilter_Within,
+  NotWhereClause,
+  OrWhereClause,
+  PossibleWhereClauseFilters,
+  WhereClause,
+} from "./aggregate/WhereClause.js";
+export { DistanceUnitMapping } from "./aggregate/WhereClause.js";
+export type { OsdkObjectPropertyType } from "./Definitions.js";
+export type {
+  AllGroupByValues,
+  GroupByClause,
+  GroupByRange,
+} from "./groupby/GroupByClause.js";
+export { DurationMapping } from "./groupby/GroupByClause.js";
```

Comment:
So much API surface :/

We definitely need to see what we can prune, either cause it just doesnt need to be exported or if we dont need all the exact types elsewhere. 

I'm actually quite glad we have a simple way to see the back compat surface, I just want to make sure its manageable.